app/assets/stylesheets/govuk_publishing_components/components/_document-list.scss

```diff
@@ -103,5 +103,67 @@
 .gem-c-document-list__highlight-text {
   @include govuk-font(16, bold);
   margin: 0 0 govuk-spacing(3) 0;
+}
+
+.gem-c-document-list__children {
+  margin-bottom: 0;
+
+  @include govuk-media-query($from: desktop) {
+    margin-left: govuk-spacing(4);
+    margin-top: govuk-spacing(4);
+
+    @supports (display: grid) {
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      column-gap: govuk-spacing(3);
+    }
+  }
+}
+
+.gem-c-document-list-child {
+  @include govuk-font($size: 16);
+
+  @include govuk-media-query($from: desktop) {
+    padding-bottom: govuk-spacing(2);
+  }
+
+  @include govuk-media-query($until: desktop) {
+    position: relative;
+    padding-left: govuk-spacing(5); //25px on mobile
+    padding-top: govuk-spacing(2);
+
+    &:before {
+      content: "—";
+      position: absolute;
+      left: 0;
+      width: 20px;
```

Comment:
Is this needed? It seems to work ok without

Reply:
Removed.